synapse/cortex.py

```diff
@@ -759,6 +761,10 @@ async def setStormVar(self, name, valu):
         self.user.confirm(('globals', 'set', name))
         return await self.cell.setStormVar(name, valu)
 
+    async def syncNodeFilteredEdits(self, offs, matchdef, wait=True):
+        self.user.confirm(('syncnodefilterededits',))
```

Comment:
I'd rather not commit ourselves to crazy specific permissions like this.  If it's really just something that's meant for admin use for now, we can just `@adminapi` mark it

Reply:
Let's discuss. I was hoping that the service wouldn't need adminapi. But, yeah this was sorta a shot in the dark.